examples/server/server.cpp

```diff
@@ -3028,13 +3028,25 @@ int main(int argc, char ** argv) {
         log_data["api_key"] = "api_key: " + std::to_string(sparams.api_keys.size()) + " keys loaded";
     }
 
-    // load the model
-    if (!ctx_server.load_model(params)) {
-        state.store(SERVER_STATE_ERROR);
-        return 1;
-    } else {
+    try {
+        // Attempt to load the model
+        if (!ctx_server.load_model(params)) {
+            LOG_ERROR("unable to load model", {{"error", "Load model failed"}});
```

Comment:
Here is why it makes no sense to me: we already catch the `unable to load model`. Why do we need another try..catch to do that?